sphinx/writers/latex.py

```diff
@@ -863,26 +867,30 @@ def visit_desc_parameterlist(self, node: Element) -> None:
         if self.has_tp_list:
             if self.orphan_tp_list:
                 # close type parameters list (#2)
-                self.body.append('}{')
+                self.body.append('}' + CR + '{')
                 # empty parameters list argument (#3)
                 return
         else:
             # close name argument (#1), open parameters list argument (#2)
-            self.body.append('}{')
+            self.body.append('}' + CR + '{')
         self._visit_sig_parameter_list(node, addnodes.desc_parameter)
 
     def depart_desc_parameterlist(self, node: Element) -> None:
         # close parameterlist, open return annotation
-        self.body.append('}{')
+        self.body.append('}' + CR + '{')
 
     def visit_desc_type_parameter_list(self, node: Element) -> None:
         # close name argument (#1), open type parameters list argument (#2)
-        self.body.append('}{')
+        self.body.append('}' + CR + '{')
         self._visit_sig_parameter_list(node, addnodes.desc_type_parameter)
 
     def depart_desc_type_parameter_list(self, node: Element) -> None:
         # close type parameters list, open parameters list argument (#3)
```

Comment:
I would put this comment in the else branch and in the if branch I would say:

```py
# Close type parameters list, skip parameters list argument, open return annotation.
# The return annotation argument context must be opened here since this node's next
# sibling is not a ``desc_parameterlist`` node (hence, ``depart_desc_parameterlist``
# will never be called).
```